conda/meta.yaml

```diff
@@ -11,15 +11,16 @@ build:
   noarch: python
   preserve_egg_dir: True
   number: 2
-  script: {{PYTHON}} setup.py install #--single-version-externally-managed --record=record.txt
+  #script: {{PYTHON}} setup.py install #--single-version-externally-managed --record=record.txt
+  script: {{ PYTHON }} -m pip install .
```

Comment:
Do we need other options  to pip install?

Reply:
I would sugger to use:

`  --no-deps                   Don't install package dependencies.`